vignettes/eatPrep.Rmd

```diff
@@ -57,38 +85,60 @@ Information/metadata about items is stored in many different ways, e.g., databas
 
 Sheets have specific names, also the columns in those sheets have specific names.
 
-Example: inputList, inputMinimal
-
 `inputMinimal` contains the bare minimum for all functions in `eatPrep` to work, `inputList` contains additional metadata about the items (such as information about the task's content).
 
+<!--Example: inputList ? -->
+
+Example: inputMinimal
+
+```{r tempsetup}
+inputMinimal <- list(units = inputList$units[ -nrow(inputList$units), c("unit", "unitAggregateRule")],
+                     subunits = inputList$subunits[, c("unit", "subunit","subunitRecoded")],
+                     values = inputList$values[ , c("subunit", "value", "valueRecode", "valueType")],
+                     unitRecodings = inputList$unitRecodings[ , c("unit", "value", "valueRecode", "valueType")],
+                     blocks = inputList$blocks,
+                     booklets = inputList$booklets,
+                     rotation = inputList$rotation)
+```
+
 ### Input tables
 Let's look at the tables from bottom to top, starting with items and their values. In the values sheet, we define values and recode values for each subitem.
 
-
 Here we see 3 items (called units) from `inputList` bzw. `inputMinimal`.
-One mc (item 1), one short response (item 2), one item with 3 subitems short response (item 12)
+One mc (item 1), one short response (item 5), one item with 3 subitems short response (item 12)
 
 ```{r inputList}
 items <- c("I01", "I05", "I12")
 subitems <- c("I01", "I05", "I12a", "I12b", "I12c")
 
 kable(inputMinimal$units[which(inputMinimal$units$unit %in% items), ])
-kable(inputMinimal$subunits[which(inputMinimal$subunits$subunit %in% subitems), ])
-kable(inputMinimal$values[which(inputMinimal$values$subunit %in% subitems), ])
-kable(inputMinimal$unitRecodings)
 
```

Comment:
which order do you want to have? because the tables were in a different order than the explanations below